src/newlines.cpp

```diff
@@ -5082,8 +5082,8 @@ void newline_after_label_colon()
 
 static bool is_class_one_liner(Chunk *pc)
 {
-   if (  (  chunk_is_token(pc, CT_FUNC_CLASS_DEF)
-         || chunk_is_token(pc, CT_FUNC_DEF))
+   if (  (  pc->Is(CT_FUNC_CLASS_DEF)
```

Comment:
needs check against nullptr

Reply:
Same question as above, I see one usage of this function where the caller checks it. Should I safeguard from any future potential wrong usage? I have no objection in doing it, just so I know for future similar cases :)

Reply:
Same answer, if the callers already check it, we can skip adding a check here since it would be temporary anyway.